scripts/schema/config/domains.json

```diff
@@ -141,5 +130,15 @@
         }
       }
     }
+  },
+  "$defs":
+  {
+    "Attributes":
+    {
+      "type": "array",
+      "additionalItems": false,
+      "items": { "type": "integer"},
+      "minItems": 1
+    }
   }
```

Comment:
If we end up with more of these shared across configs, might be worth bundling into a common/utilities section rather than copy paste. With only one duplicate though, not worth doing yet.